arduino-ide-extension/src/node/core-service-impl.ts

```diff
@@ -39,6 +40,8 @@ export class CoreServiceImpl extends CoreClientAware implements CoreService {
 
   protected uploading = false;
 
+  private FLUSH_OUTPUT_MESSAGES_TIMEOUT_MS = 32;
```

Comment:
Missing readonly? If this is a static constant, it should be `static`. Or please move out from the class to the module scope and use `const`. Having a variable in the class with `SUCH_NAMING` is confusing. Thank you!